### PR TITLE
Translate cloudevents/spec/blob/main/cloudevents/primer.md to zh-CN #982

### PR DESCRIPTION
**Pull request by sunshengjing**
Signed-off-by: Sun Shengjing <sunshengjing@Suns-MacBook-Pro.local>

### CODE REVIEW

File: cloudevents/languages/zh-CN/primer.md [hunk 1]
```diff
@@ -149,22 +149,22 @@ CloudEvents 规范目前不强制或提倡任何有关授权、数据完整性
 CloudEvents 规范集定义了四种有助于形成分层架构模型的不同类型的协议元素。
 
 1. [基本规范](spec.md) 定义了一个抽象信息模型，
-   该模型由属性（键值对）和构成 CloudEvent 的相关规则组成。
+   该模型由属性（键值对）和构成 CloudEvent 的相关规则组成。此规范包含了*核心属性*的定义。有些核心属性是必须出现在所有的 CloudEvents 中，有些则是可选的。
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/primer.md:152]:
```suggestion
   该模型由属性（键值对）和构成 CloudEvent 的相关规则组成。此规范包含了*核心属性*的定义。有些核心属性必须出现在所有的 CloudEvents 中，有些则是可选的。
```

File: cloudevents/languages/zh-CN/primer.md [hunk 2]
```diff
@@ -149,22 +149,22 @@ CloudEvents 规范目前不强制或提倡任何有关授权、数据完整性
...
    以将其映射到应用程序协议的头部和负载元素。
 4. 协议绑定, 如. [HTTP协议绑定](../../bindings/http-protocol-binding.md),
    在HTTP to HTTP的情况下，
-   定义了 CloudEvent 如何绑定到应用程序协议的传输层 。
+   定义了 CloudEvent 如何绑定到应用程序协议的传输层。
    协议绑定不限制传输层的使用方式，这意味着 HTTP绑定可用于任何 HTTP方法以及请求和响应消息。
 
-为了确保更广泛的互操作性，CloudEvents规范集为使用专有应用协议的事件传输提供了特定约束。
+为了确保更广泛的互操作性，CloudEvents 规范集为使用专有应用协议的事件传输提供了特定约束。
 [HTTP Webhook](../../http-webhook.md)规范并非特定于 CloudEvents，
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/primer.md:163]:
```suggestion
[HTTP Webhook](../../http-webhook.md) 规范并非特定于 CloudEvents，
```

File: cloudevents/languages/zh-CN/primer.md [hunk 3]
```diff
@@ -226,9 +226,9 @@ CloudEvents 规范不强制要求要使用的特定模式，甚至不要求必
 当 CloudEvent 的数据以向后不兼容的方式更改时，`dataschema` 属性的值通常应更改，
 就像上述的 `type` 属性的情况一样。
 
-## CloudEvent Attributes/属性
+## CloudEvent Core Attributes/核心属性
 
-本节介绍了与CloudEvent 属性相关的其它背景和设计要点。
+本节介绍了与CloudEvent核心属性相关的其它背景和设计要点。
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/primer.md:231]:
```suggestion
本节介绍了与 CloudEvent 核心属性相关的其它背景和设计要点。
```

File: cloudevents/languages/zh-CN/primer.md [hunk 4]
```diff
@@ -240,7 +240,7 @@ CloudEvents 规范不强制要求要使用的特定模式，甚至不要求必
 因为这超出了本规范的范围。
 唯一的例外是如果支持事件的重播，在这些情况下，可以使用 id 来检测这一点。
 
-由于一次事件的发生可能导致生成多个cloud event，
+由于一次事件的发生可能导致生成多个 cloud event，
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/primer.md:243]:
```suggestion
由于一次事件的发生可能导致生成多个事件，
```

**Review by JieDing — COMMENTED** [review]
Thx for your efforts. Here are some feedbacks. @sunshengjing

File: cloudevents/languages/zh-CN/primer.md (hunk 1 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/primer.md:152]: revised.

File: cloudevents/languages/zh-CN/primer.md (hunk 2 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/primer.md:163]: space added.

File: cloudevents/languages/zh-CN/primer.md (hunk 3 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/primer.md:231]: space added

File: cloudevents/languages/zh-CN/primer.md (hunk 4 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/primer.md:243]: modified. thanks for pointing out.

**Comment by JieDing** [comment]
@sunshengjing Thanks for your efforts again. 

@duglin LGTM

File: cloudevents/languages/zh-CN/primer.md [hunk 5]
```diff
@@ -665,7 +666,7 @@ CloudEvents 将为任何事件使用者（例如serverless平台）提供一种
 }
 ```
 
-#### AWS - CloudWatch Events
+#### AWS - CloudWatch Events/亚马逊-CloudWatch 事件
```
Comment (duglin) [inline cloudevents/languages/zh-CN/primer.md:669]: very minor thing, but can we put spaces before/after the "-" here and on line 692 ?

**Comment by duglin** [comment]
Please let me know when you're done with all edits.

File: cloudevents/languages/zh-CN/primer.md (hunk 5 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/primer.md:669]: sure. I will add them

**Comment by sunshengjing** [comment]
> Please let me know when you're done with all edits.

I fixed all, thanks.

**Comment by duglin** [comment]
It looks like the "verify" errors are real

**Comment by sunshengjing** [comment]
> It looks like the "verify" errors are real

we have fixed them, thanks

**Comment by duglin** [comment]
Thanks. Are all edits done and it’s ready to merge?

**Comment by duglin** [comment]
I'm going to assume all edits are done - Approved on the 4/28 call, so merging

**Pull request merged**